packages/@o3r/styling/scss/utils/_functions.scss

```diff
@@ -74,7 +74,9 @@ $test-mode-enabled: false !default;
 /// @access private
 @function log-variable($name, $value: null, $details: null) {
   $metadata-debug: false !default;
-  @if (meta.function-exists('metadata-report')) {
+  $is-internal-property: string.index($name, '_') == 1;
+
+  @if (meta.function-exists('metadata-report') and not $is-internal-property) {
```

Comment:
Are you sure to ignore "private" variable from the extration?
As they can be referred by others, it can not produce inconsistencies in the metadata?